server/src/main/java/io/druid/server/coordinator/CoordinatorStats.java

```diff
@@ -33,74 +35,103 @@
   private final Map<String, Object2LongOpenHashMap<String>> perTierStats;
   private final Object2LongOpenHashMap<String> globalStats;
 
-  private static class DefaultingObject2LongOpenHashMap<K>
-      extends Object2LongOpenHashMap<K>
+  public CoordinatorStats()
   {
-    // to behave like DefaultingHashMap
-    @Override
-    public Long get(
-        final Object key
-    )
-    {
-      return getLong(key);
-    }
+    perTierStats = Maps.newHashMap();
+    globalStats = new Object2LongOpenHashMap<>();
   }
 
-  public CoordinatorStats()
+  public boolean hasPerTierStats(
+  )
   {
-    perTierStats = Maps.newHashMap();
-    globalStats = new DefaultingObject2LongOpenHashMap<>();
+    return !perTierStats.isEmpty();
   }
 
-  public Map<String, ? extends Object2LongMap<String>> getPerTierStats()
+  public Set<String> getTiers(
+      final String statName
```

Comment:
There is no rule in Druid code style that params must go on the next line; if the method signature with all params fits 120 characters (line size limit), then it should go on a single line.